commcare_connect/templates/account/login.html

```
@@ -64,22 +65,25 @@ <h6 class="title text-brand-deep-purple">Login</h6>
-                 alt="Compact Logo"
-                 class="w-full">
-          </div>
-          <span class="text-sm">Login with CommCareHQ</span>
-        </button>
-      </form>
-    </div>
+    {% configured_provider "commcarehq" as commcarehq %}
+    {% if commcarehq %}
+      <span class="text-sm text-gray-400 text-center">or</span>
+      <div class="space-y-2 place-items-center">
+        <form method="post"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)